dotnet/csharp-grpc-load/load/LoadGenerator.cs

```diff
@@ -181,6 +202,16 @@ private async Task<Stats> RunOneSet(CacheClient.Scs.ScsClient client, Stats stat
         await Set.Get(set, elementStart, count, client, util, stats, cancellationToken);
         return stats;
     }
+    private async Task<Stats> RunOneList(Scs.ScsClient client, Stats stats, ulong i, Random random, CancellationToken cancellationToken)
+    {
+        uint list = (uint)random.NextInt64(keyspace);
```

Comment:
Are we using the same list name here as what we are creating at the start of the test? Same for the `ClientList` mode.

Reply:
with a keyspace of 1 we are